blueprint/core/blueprint_AppHarness.cpp

```diff
@@ -52,22 +56,27 @@ namespace blueprint
                     }
                 }
 
-                if (onAfterEach) { onAfterEach(f); }
+                if (onAfterEach)
+                {
+                    onAfterEach(f);
+                }
             }
 
-            if (onAfterAll) { onAfterAll(); }
+            if (onAfterAll)
+            {
+                onAfterAll();
+            }
         });
-
     }
 
     //==============================================================================
-    void AppHarness::watch (const juce::File& f)
+    void AppHarness::watch(const juce::File& f)
```

Comment:
Are we trying to stick to the juce style guide?  If so I think non-empty parens would have a preceding space (i.e. `SpaceBeforeParens: NonEmptyParentheses` instead of `SpaceBeforeParens: ControlStatements`)

Reply:
Heh. Hopefully we don't need to fully comply .... I hate that leading space.

Reply:
Haha, such a can of worms this business.  I'm in the not caring so long as a tool is doing it for me camp.

Reply:
Hah, I'll admit, I'm also not a fan of that leading space. Generally though, I too am in the "i don't care let's just standardize it" camp.

I think generally if we can reach a small consensus of "yea looks good enough for me" then we should go for it